api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -111,11 +111,13 @@ public void syncBillingProjectStatus() {
                 + entry.getFireCloudProjectName()
                 + ". Project not found.",
             e);
+        entry.setStatusEnum(ERROR, this::getCurrentTimestamp);
```

Comment:
The previous behavior was intentional, see https://pmi-engteam.slack.com/archives/CHRN2R51N/p1570652916047400 ; if a project fails repeatedly this way, it will eventually transition into the error state after an hour by the other cron process (pending CREATING state for too long).

What issue are you trying to solve?

Reply:
This is not true in Perf - see the billing buffer there

Reply:
Ah.. cleanBillingBuffer should probably run much more frequently, it's set to run every 24h currently